kernel/shared/src/main/scala/cats/effect/kernel/Resource.scala

```diff
@@ -1330,10 +1354,19 @@ abstract private[effect] class ResourceFOInstances1 {
 }
 
 abstract private[effect] class ResourceMonadCancel[F[_]]
-    extends ResourceMonadError[F, Throwable]
+    extends ResourceMonad[F]
     with MonadCancel[Resource[F, *], Throwable] {
   implicit protected def F: MonadCancel[F, Throwable]
 
+  override def attempt[A](fa: Resource[F, A]): Resource[F, Either[Throwable, A]] =
+    fa.attempt(F)
```

Comment:
Unless it's necessary for some reason? 🤔 

```suggestion
    fa.attempt
```